Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/NMDataServiceTest.cs

```diff
@@ -308,6 +308,36 @@ public void WhenDownloadFssFileAsyncThrowsException_ThenShouldExecuteCatch()
             Assert.IsTrue(result.IsFaulted);
         }
 
+        [Test]
+        public void WhenDownloadFSSZipFileAsyncIsCalled_ThenShouldReturnByteArray()
```

Comment:
you need to be more descriptive in the test tite and make it clearer as to what the test is actually targeting

Reply:
Renamed test case as : _WhenDownloadFSSZipFileAsyncIsCalled_ThenShouldReturnFileByteArray_